app/packs/src/stores/alt/stores/UIStore.js

```diff
@@ -188,31 +188,29 @@ class UIStore {
   handleCheckAllElements(params) {
     this.waitFor(ElementStore.dispatchToken);
 
-    let { type, range } = params;
-    let { elements } = ElementStore.getState();
+    const { type, range } = params;
+    const { elements } = ElementStore.getState();
 
-    if (range == 'all') {
-      if (this.state.currentSearchSelection && elements[type + "s"].ids) {
```

Comment:
relevant change

Reply:
Just to be sure, the only behavioral change in this file's diff is the removal of `this.state.currentSearchSelection` from the guard clause? For the future please change behavior and formatting in separate commits.